code_share/app_code_share/templates/app_code_share/code_view.html

```diff
@@ -4,6 +4,17 @@
 
 {% block css %}
   <link rel="stylesheet" href="{% static 'css/codeshare.css' %}">
+  
+  <style type="text/css">
+    .CodeMirror {
+      border: 1px solid #eee;
+      height: 70%;
+      width: 100%;
+      line-height: 1em;
+      font: 1.5rem sans-serif, sans-serif;
+      _position: relative;
+    }
+  </style>
```

Comment:
There was an issue as editor was not initialised before applying that css property. Now it's fixed.
Nice resources as to [how browsers works](https://www.html5rocks.com/en/tutorials/internals/howbrowserswork/).